packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -80,25 +80,25 @@ export default function Sidebar({
           icon: <ListIcon Icon={HotIcon} />,
           title: 'Popular',
           path: '/popular',
-          action: () => onNavTabClick('popular'),
+          action: onNavTabClick ? () => onNavTabClick('popular') : null,
```

Comment:
This might look better. What do you think?

```suggestion
          action: () => onNavTabClick.?('popular'),
```